packages/dashboard/config/index.ts

```diff
@@ -17,6 +17,14 @@ export const config: any = {
   ethereumRpcUrl: process.env.ETHEREUM_RPC_URL,
   ethereumMnemonic: process.env.ETHEREUM_MNEMONIC,
   elementAnchorContract: process.env.ELEMENT_ANCHOR_CONTRACT,
+  awsCredentials: {
```

Comment:
hopefully only required when you want to use photon.

Reply:
Yes it won't break anything it not set and pointing to another did method, there is a section at the bottom of this file that sorts this out https://github.com/transmute-industries/sidetree.js/pull/360/files#diff-46d19c00119f9d4f400b19637eaec73c452e748f2e7cf6ce2af8c22c8341dd4fR91